backend/private-graph/graph/middleware.go

```diff
@@ -29,7 +29,7 @@ var (
 	workspaceTokenHandler APITokenHandler
 )
 
-var HighlightAdminEmailDomains = []string{"@highlight.run", "@highlight.io"}
+var HighlightAdminEmailDomains = []string{"@highlight.io"}
```

Comment:
do we have to get rid of `highlight.run` here? I've always been using that account

Reply:
my bad, will pull that out into a separate PR (motivation is that if we ever stop paying for our highlight.run DNS, this could be problematic)